src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -381,7 +386,7 @@ def register_dataset_udf(
         *args,
         name=f"{dag_name} ingestor ",
         expand_node_output=input_list_node,
-        resources=DEFAULT_RESOURCES if resources is None else resources,
+        resources=resources or ingest_resources or DEFAULT_RESOURCES,
```

Comment:
@ktsitsi Users of `ingest(resources={...})` without the `as_batch()` wrapper will get the behavior they expect. At the same time, our UI can specify `ingest_resources` and `as_batch()` will pass it through. How does that sound? I'll make the corresponding change upstream in the UI repo.